src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIBuildJobExecutionService.java

```diff
@@ -269,33 +271,68 @@ private LocalCIBuildResult runScriptAndParseResults(ProgrammingExerciseParticipa
 
     // --- Helper methods ----
 
-    private LocalCIBuildResult parseTestResults(TarArchiveInputStream testResultsTarInputStream, String assignmentRepoBranchName, String assignmentRepoCommitHash,
+    private List<String> getTestResultPath(ProgrammingExercise programmingExercise) {
+        List<String> testResultPaths = new ArrayList<>();
+        switch (programmingExercise.getProgrammingLanguage()) {
+            case JAVA, KOTLIN -> {
+                if (ProjectType.isMavenProject(programmingExercise.getProjectType())) {
+                    if (!programmingExercise.hasSequentialTestRuns()) {
```

Comment:
Why invert this expression? Guess just `if(programmingExercise.hasSequentialTestRuns())` is easier to read